state/src/storage/tx/node_metadata.rs

```diff
@@ -22,6 +26,11 @@ const CLUSTER_ID_KEY: &str = "cluster-id";
 const NODE_KEYPAIR_KEY: &str = "node-keypair";
 /// The name of the local peer addr in the node metadata table
 const LOCAL_ADDR_KEY: &str = "local-addr";
+/// The key for the local relayer's wallet ID in the node metadata table
+const LOCAL_WALLET_ID_KEY: &str = "local-wallet-id";
+/// The key for the local relayer's fee decryption key in the node metadata
+/// table
+const LOCAL_RELAYER_FEE_KEY: &str = "local-relayer-fee";
```

Comment:
tiny nit can we have the key be `"local-relayer-fee-key"`

Reply:
did this in the follow up!